platoon/channel/worker.py

```diff
@@ -285,7 +285,7 @@ def _register_to_platoon(self):
             response = self.send_req("platoon-get_platoon_info",
                                      info={'device': self.device,
                                            'local_id': lid})
-            nlid = base64.b64decode(response['local_id'].encode('ascii'))
+            nlid = base64.b64decode(str(lid).encode('ascii'))
```

Comment:
This does not use the local_id that we got from the server so it will probably get this wrong for more than one GPU.